pkg/operator/validations.go

```diff
@@ -39,22 +39,10 @@ func ValidateDurationField(durationField string) error {
 
 // CompareScrapeTimeoutToScrapeInterval validates value of scrapeTimeout based on scrapeInterval
 func CompareScrapeTimeoutToScrapeInterval(scrapeTimeout, scrapeInterval monitoringv1.Duration) error {
-	var err error
 	var si, st model.Duration
 
-	// since default scrapeInterval set by operator is 30s
-	if scrapeInterval == "" {
```

Comment:
nit: Not sure if it's worth maintaining this default so that we don't have to type it for all the tests.

Reply:
It's due to limitation of unit test we had to set that since it cannot know default set by apiserver. But I think removing that check from code helps to bring more clarity :)

Reply:
Got it, thanks for clarifying